cmake/PACE_Docs.cmake

```diff
@@ -68,6 +90,10 @@ else()
     COMMENT "Docs require sphinx and sphinx-rtd-theme to build"
     )
 
+  add_custom_target(docs-pack
+    COMMENT "Docs require sphinx and sphinx-rtd-theme to build"
+    )
```

Comment:
Looks like a duplicate of the next block? As it isn't - docs-pack rather than manual, may be worth commenting what the difference is

Reply:
This is the error message which is reported if you don't have the required packages and try to build. Because docs-pack is explicitly just "build the HTML docs and pack them (in tar or zip), I would say the error message is more helpful like this. Manual, however, is building a latex document

Reply:
I was more thinking that when a maintainer comes along sometime in the future and has to distinguish between these things, some cue to help them along could be good. You could do it as a Groovy comment outside the add-custom-target block, or you could just extent the COMMENTs slightly with "Latex docs require .... " and "HTML docs require..."

Reply:
I can't do it as a Groovy comment, as this is CMake, but I have extended the comments to clarify what is being built.